python_modules/dagster/dagster/_core/definitions/asset_job.py

```diff
@@ -27,7 +25,9 @@
 from dagster._core.definitions.policy import RetryPolicy
 from dagster._core.errors import DagsterInvalidDefinitionError, DagsterInvalidSubsetError
 from dagster._core.selector.subset_selector import AssetSelectionData
+from dagster._core.utils import toposort
 from dagster._utils.merger import merge_dicts
+from dagster._vendored.toposort.toposort import CircularDependencyError
```

Comment:
what's the benefit of having the core/utils import? I think its good to be explicit when we are vendoring and the two different import paths for the same underlying package is a little unsightly

Reply:
Actually I want to keep it until the workaround is deleted. I will add another PR to this stack.